build/BUILD

```diff
@@ -160,3 +171,31 @@ filegroup(
         "//test/kubemark:all-srcs",
     ],
 )
+
+# These bundles group images at no build time cost unless a .tar is built from
+# the bundle. We use these bundles to push all images from the pushall rule.
+docker_bundle(
+    name = "all_images",
+    images = {
+        "gcr.io/google_containers/%s:{STABLE_DOCKER_TAG}" % (binary): binary + "-internal"
+        for binary in DOCKERIZED_BINARIES.keys()
+    },
+    stamp = True,
+)
+
+docker_bundle(
+    name = "all_dev_images",
+    images = {
+        "gcr.io/kubernetes-ci-images/bazel/%s:{STABLE_DOCKER_TAG}" % (binary): binary + "-internal"
```

Comment:
would it make more sense to use a "make variable" for the repo? I guess we don't have a good way of setting defaults, though.